Source/ARTHttp.m

```diff
@@ -175,18 +179,19 @@ - (void)executeRequest:(NSMutableURLRequest *)request completion:(void (^)(NSHTT
     [self.logger debug:@"%@ %@", request.HTTPMethod, request.URL.absoluteString];
     [self.logger verbose:@"Headers %@", request.allHTTPHeaderFields];
 
+    __weak typeof(self) weakSelf = self;
```

Comment:
@tcard When the reference count is 0, the REST will live until the a HTTP request is active. The same goes for the Realtime. It will live until a connection is active.